ansible/roles/ansible-tvault-contego-extension/templates/nova_compte_extension.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+xtrace=$(set +o | grep xtrace)
+set +o xtrace
+
+. {{virtual_env_path}}/bin/activate
+
+file="{{NOVA_COMPUTE_FILTERS_FILE}}"
+section="Filters"
+option="qemu-img: EnvFilter"
+remove="yes"
+other="yes"
+line=$(sed -ne "/^\[$section\]/,/^\[.*\]/ { /^$option[ \t]*=/ p; }" "$file")
+if [ "$other" = "yes" ] && [ "$remove" = "yes" ]; then
+   line=$(sed -ne "/^\[$section\]/,/^\[.*\]/ { /^$option[ \t]*,/ p; }" "$file")
+fi
+#$xtrace
+echo "$line"
+if [ "$remove" = "yes" ] && [ ! -z "$line" ]; then
+    grep -v "$line" "$file" > "$file.bak"
+    mv "$file.bak" "$file"
+fi
```

Comment:
what are we doing with this file ??

Reply:
its looking "qemu-img: Env Filter"  under Filters section.